packages/manager/modules/support/src/modal/modal.constants.js

```diff
@@ -0,0 +1,28 @@
+const helpRoot = 'https://help-beta.ovhcloud.com/csm';
+const homeIndex = '-home?id=csm_index';
+
+export const GUIDES_LINK = {
+  DE: `${helpRoot}/de${homeIndex}`,
```

Comment:
it's a good practice to have a default entry (en for example).
```suggestion
  DEFAULT: `${helpRoot}/en${homeIndex}`,
  DE: `${helpRoot}/de${homeIndex}`,
```